view/mail-body.html

```diff
@@ -1,749 +1,119 @@
-<!doctype html>
-<html xmlns="http://www.w3.org/1999/xhtml" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:o="urn:schemas-microsoft-com:office:office">
```

Comment:
What is the reason for omitting the XML namespace attributes for MS Office?

Reply:
no xml formats are used in the email, and it is not generated through outlook - it can be kept however. the email might lose formatting when responded to through outlook